modules/auth/auth.constants.js

```diff
@@ -0,0 +1,130 @@
+// Package imports
+const env = require('dotenv').config();
+
+module.exports = {
+  responseMessages: {
+    authorizedUser: {
+      failure: 'You are not authorized. Please login first',
+    },
+    logOutUser: {
+      success: 'You are logged out successfully.',
+      failure: 'Failed logging out. Please try again!',
+    },
+    changePassword: {
+      wrongCredentails: 'Wrong Crendentials. Please try again!',
+      conflict: 'You are signed in through google/github.',
+      success: 'Your password is changed successfully',
+      failure: 'Failed to change password. Please login again and then try!',
+    },
+    generalErrorMessage: 'Something went wrong! Try again!',
+    forgotPassword: {
+      resetPassword: {
+        success: 'Your password has been reset successfully',
+        failure: 'Failed to reset password. Please try again!',
+        tokenExpiredMessage: 'Your token has expired. Please try again',
+        wrongCredentails: 'Wrong Crendentials. Please try again!',
+      },
+      resetPasswordEmail: {
+        success: 'Password reset mail is sent to your account',
+        failure: 'Password reset mail could not be sent! Check your network connectivity!',
```

Comment:
How do you know it's a connectivity issue?
The message should be generic.

Reply:
done